README.md

```
@@ -57,6 +70,18 @@ flowchart TD
 
 ---
 
+## 📁 Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)